### PR TITLE
Moved functions from function files into appropriate scripts : Caleb

### LINKED ISSUE #205: Consolidate astronomy pipeline files
As a developer, I want the files to be in a similar structure across directories, so that the file structure is easier to understand and work with, especially with testing.

### PR DESCRIPTION
**Pull request by CalebMathers**
This closes #205 

- Moved the functions and import from the `..._functions.py` files into their main scripts. Made sure module imports in other files are correct and tests still pass.
- Fixed a few pylint issues.
- Changed Dockerfile to only copy the necessary files now.

### CODE REVIEW

File: astronomy-pipeline/astronomy_extract.py [hunk 1]
```diff
@@ -1,10 +1,189 @@
...
+
+from os import environ as ENV
 from datetime import date, timedelta, datetime
+import base64
 import json
 
-from astronomy_extract_functions import get_db_regions, get_position_data, get_moon_urls
-from astronomy_extract_functions import fill_region_time_dict
+import requests
+from dotenv import load_dotenv
```
Comment (AlexOBarnes) [inline astronomy-pipeline/astronomy_extract.py:12]: Much more consistent with our other files

File: astronomy-pipeline/Dockerfile [hunk 2]
```diff
@@ -7,9 +7,7 @@ RUN pip install -r requirements.txt
 
 COPY astronomy_load.py .
 COPY astronomy_extract.py .
-COPY astronomy_extract_functions.py .
 COPY astronomy_transform.py .
-COPY astronomy_transform_functions.py .
```
Comment (AlexOBarnes) [inline astronomy-pipeline/Dockerfile:12]: Good idea

File: astronomy-pipeline/astronomy_transform.py [hunk 3]
```diff
@@ -1,12 +1,134 @@
-"""
-astronomy_transform
-Transform script to reshape Astronomy API data into a format
-for upload to the RDS database.
-"""
-
-from astronomy_transform_functions import (get_data_into_dataframe, clean_position_data,
-                                           get_moon_df, convert_positions_datetime,
-                                           convert_moon_datetime)
+"""Transform script to reshape Astronomy API data into a format
```
Comment (AlexOBarnes) [inline astronomy-pipeline/astronomy_transform.py:1]: very consistent

**Review by AlexOBarnes — APPROVED** [review]

**Review by olliejth — APPROVED** [review]
Much tidier.

**Pull request merged**